pkg/resource/grafana-agent-config/grafana-agent-config.go

```diff
@@ -93,6 +105,10 @@ loki.write "default" {
 }`,
 				},
 			},
+			Controller: controller{
+				Replicas: 1,
+				Type:     deployment,
```

Comment:
Don't we decide to use sts ?

Reply:
We decided not to on friday when you cut the call :D

Reply:
Can you add a comment explaining why we decided to use a deployment with 1 replica here or in the linked ticket?

Reply:
Sure, we decided to use a deployment because statefulsets should be used for storing metrics (when using a wal) and we are fine with losing a few events when the pod restarts. We're mostly concerned about creating a PV on all WCs